gspread_formatting/models.py

```diff
@@ -187,11 +189,29 @@ def __init__(self, themeColor=None, rgbColor=None):
 class Color(CellFormatComponent):
     _FIELDS = ('red', 'green', 'blue', 'alpha')
 
-    def __init__(self, red=None, green=None, blue=None, alpha=None):
+    def __init__(self, red=None, green=None, blue=None, alpha=None, hexcolor=None):
         self.red = red
         self.green = green
         self.blue = blue
         self.alpha = alpha
+        if hexcolor:
+            self.fromHex(hexcolor)
+
+    def toHex(self):
+        # Dont worry about alpha channel yet
```

Comment:
Hex encoding of RGBA colors does specify how to include alpha channel: if alpha value is present, convert from 0.0-1.0 range to 0-255 range and encode as two hex digits after the two hex digits for blue. e.g. `#80FFFF80`.  If alpha value is absent, its two hex digits should be omitted.